ROOTFS/usr/bin/waggle-network-watchdog

```diff
@@ -80,7 +70,9 @@ def reboot_os():
     # aggressively but safely reboot the system
     subprocess.run(["systemctl", "--force", "reboot"])
 
-ssh_ok_file = must_read_config_file("/etc/waggle/config.ini")
+config = must_read_config_file("/etc/waggle/config.ini")
+ssh_ok_file = config["watchdog"]["ssh_ok_file"]
```

Comment:
if the fields `watchdog` and/or `ssh_ok_file` are not present in config this will throw an exception and crash right?

lets protect against that and have `ssh_ok_file` be set to `None` in a failure condition such that this service still operates and attempts any recovery operations.

Reply:
Fair point. I just committed a fix to gracefully return as much config was available and am re-handling the default case of there not being an ssh_ok_file.